docs/modules/ROOT/pages/tutorial/deploy.adoc

```diff
@@ -57,10 +57,40 @@ image::tutorial-deploy-step4-wizard.png[Deploy block end wizard]
 
 NOTE: You configured the test environment to learn without the risk of losing actual funds. The steps are the same to set up a production environment.
 
-[[hardhat-integration]]
-=== Hardhat Integration
+Defender supports both Hardhat and Foundry integrations. Pick the one that suits your project!
 
-Deploying through Defender is extremely easy with Hardhat. Once it's installed, follow these steps to create a new directory and project:
+[[foundry-setup]]
+=== Foundry setup
+
+First, make sure you have https://book.getfoundry.sh/getting-started/installation[Foundry, window=_blank] installed. Follow these steps to create a new directory and project:
+
+. Run the this command in your terminal:
++
+```
+forge init deploy-tutorial && cd deploy-tutorial && forge install OpenZeppelin/openzeppelin-foundry-upgrades --no-commit && forge install OpenZeppelin/openzeppelin-contracts-upgradeable --no-commit
```

Comment:
This is a recent addition to the recommended steps, to ensure the user gets the latest version of `forge-std`:
```suggestion
forge init deploy-tutorial && cd deploy-tutorial && forge install foundry-rs/forge-std --no-commit && forge install OpenZeppelin/openzeppelin-foundry-upgrades --no-commit && forge install OpenZeppelin/openzeppelin-contracts-upgradeable --no-commit
```